test/GoogleMaps.js

```diff
@@ -108,6 +108,13 @@
 				expect(GoogleMapsLoader.createUrl()).to.be.equal(baseUrl + '?callback=' + cb + '&sensor=false&channel=abcdefghijkl');
 			});
 
+			it('should create url with language', function() {
+				GoogleMapsLoader.CHANNEL = undefined;
```

Comment:
Without it, the setting for CHANNEL from the previous test was still applied. Do you think I missed something elsewhere?